JobConfig/digitize/OnSpill_epilog.fcl

```diff
@@ -0,0 +1,24 @@
+# set the event timing for OnSpill
+physics.producers.EWMProducer.SpillType : 1
+# this next produces additional hit-level payload to support MC truth matching.
+# It produces functionally equivalent output as the standard module but is slower, so don't use this script for timing studies
+physics.producers.TTmakeSH.FilterHits : false
+physics.producers.TTmakePH.TestFlag : true
+physics.producers.TTmakePH.StrawHitSelectionBits : ["EnergySelection","TimeSelection","RadiusSelection"]
+physics.producers.TTmakePH.StrawHitMask          : ["Dead"]
```

Comment:
Should these selection bits be defined within `Mu2e/mu2e_trig_config` to avoid needing to update multiple locations if this list changes ? Or are these the only bits that `TTmakeSH` can reasonably filter out?

Reply:
Those bits are currently defined in TrkHitReco prolog. I might be able to steal them from there.